### PR TITLE
Fix incorrect app title, package and file names

### LINKED ISSUE #80: Fix title of app when it launches (Address App -> SoConnect)
image

### LINKED ISSUE #83: Fix incorrect package name and file name
Rename the relevant packages and files from `addressBook` to `SoConnect`.

### PR DESCRIPTION
**Pull request by Ugholaf**
# Changes
- Rename the title of the app to SoConnect.
- Rename the `seedu/address` package to `soconnect`.
- Rename the relevant files, methods and attributes to match our product name - `SoConnect`.
- Fix some of the JavaDocs comments to follow the [Java Coding Standards ](https://se-education.org/guides/conventions/java/index.html).

### CODE REVIEW

File: src/main/java/soconnect/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -26,6 +27,7 @@ public class ParserUtil {
     /**
      * Parses {@code oneBasedIndex} into an {@code Index} and returns it. Leading and trailing whitespaces will be
      * trimmed.
+     *
      * @throws ParseException if the specified index is invalid (not non-zero unsigned integer).
```
Comment (PangKuangWei) [inline src/main/java/soconnect/logic/parser/ParserUtil.java]:
```suggestion
     * @throws ParseException If the specified index is invalid (not non-zero unsigned integer).
```

File: src/main/java/soconnect/logic/parser/CustomiseCommandParser.java [hunk 2]
```diff
@@ -52,29 +49,29 @@ public CustomiseCommand parse(String args) throws ParseException {
     /**
      * Converts the input into an array of Attribute objects in the required order.
      *
-     * @param args an array of argument inputs
-     * @return an array of Attribute
-     * @throws ParseException if the user input does not conform the expected format
+     * @param args an array of argument inputs.
+     * @return an array of Attribute.
+     * @throws ParseException if the user input does not conform the expected format.
```
Comment (PangKuangWei) [inline src/main/java/soconnect/logic/parser/CustomiseCommandParser.java]:
```suggestion
     * @throws ParseException If the user input does not conform the expected format.
```

File: src/main/java/soconnect/logic/parser/CustomiseCommandParser.java [hunk 3]
```diff
@@ -89,26 +86,27 @@ private Attribute[] toAttributeOrder(String[] args) throws ParseException {
...
      *
-     * @param attributeArr the input Attribute array
+     * @param attributeArr the input Attribute array.
      * @param isAttributeUsed the boolean array that index 0 represents ADDRESS, index 1 represents EMAIL,
-     *                        index 2 represents PHONE and index 3 represents TAGS
-     * @param noOfMissing the number of attributes that have to be filled in
-     * @return the Attribute array order
+     *                        index 2 represents PHONE and index 3 represents TAGS.
+     * @param noOfMissing the number of attributes that have to be filled in.
+     * @return the Attribute array order.
```
Comment (PangKuangWei) [inline src/main/java/soconnect/logic/parser/CustomiseCommandParser.java]:
```suggestion
     * @param noOfMissing The number of attributes that have to be filled in.
     * @return The Attribute array order.
```

File: src/main/java/soconnect/logic/parser/CustomiseCommandParser.java [hunk 4]
```diff
@@ -89,26 +86,27 @@ private Attribute[] toAttributeOrder(String[] args) throws ParseException {
     /**
      * Fills in the unspecified attributes in default order at the back.
      *
-     * @param attributeArr the input Attribute array
+     * @param attributeArr the input Attribute array.
      * @param isAttributeUsed the boolean array that index 0 represents ADDRESS, index 1 represents EMAIL,
```
Comment (PangKuangWei) [inline src/main/java/soconnect/logic/parser/CustomiseCommandParser.java]:
```suggestion
     * @param attributeArr The input Attribute array.
     * @param isAttributeUsed The boolean array that index 0 represents ADDRESS, index 1 represents EMAIL,
```

File: src/main/java/soconnect/logic/parser/CustomiseCommandParser.java [hunk 5]
```diff
@@ -52,29 +49,29 @@ public CustomiseCommand parse(String args) throws ParseException {
     /**
      * Converts the input into an array of Attribute objects in the required order.
      *
-     * @param args an array of argument inputs
-     * @return an array of Attribute
-     * @throws ParseException if the user input does not conform the expected format
+     * @param args an array of argument inputs.
+     * @return an array of Attribute.
```
Comment (PangKuangWei) [inline src/main/java/soconnect/logic/parser/CustomiseCommandParser.java]:
```suggestion
     * @param args An array of argument inputs.
     * @return An array of Attribute.
```

File: src/main/java/soconnect/logic/parser/Parser.java [hunk 6]
```diff
@@ -10,7 +10,8 @@
 
     /**
      * Parses {@code userInput} into a command and returns it.
-     * @throws ParseException if {@code userInput} does not conform the expected format
+     *
+     * @throws ParseException if {@code userInput} does not conform the expected format.
```
Comment (PangKuangWei) [inline src/main/java/soconnect/logic/parser/Parser.java]:
```suggestion
     * @throws ParseException If {@code userInput} does not conform the expected format.
```

**Review by PangKuangWei — CHANGES_REQUESTED** [review]
LGTM. Good job on the renaming. I have added suggestions on some capitalization that was missed.

**Review by PangKuangWei — APPROVED** [review]

File: src/main/java/soconnect/logic/parser/ParserUtil.java (hunk 1 above)
Comment (Ugholaf) [reply src/main/java/soconnect/logic/parser/ParserUtil.java]: Thanks for reviewing, I have made the changes!

**Pull request merged**